hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/EncryptedFileSystemTest.java

```diff
@@ -359,4 +361,61 @@ public void testDelete_fileAlreadyDeleted() throws IOException {
         assertFalse(efs.delete(path, false));
         assertThat(keyStore.get(path.toString()), is(nullValue()));
     }
+
+    @Test
+    public void testExists() throws IOException {
+        mockedEfs.exists(path);
+
+        verify(mockFs).exists(path);
+    }
+
+    @Test
+    public void testListStatus() throws IOException {
+        mockedEfs.listStatus(path);
+
```

Comment:
it's a bit fanatic but I like grouping test methods like:
```
setup (initialization, mocking, etc)

run test methods

assertions
```

can remove since these are so short but it's how I've done it everywhere: https://github.com/palantir/hadoop-crypto/blob/develop/hadoop-crypto/src/test/java/com/palantir/crypto2/hadoop/PathConvertingFileSystemTest.java